src/frontend/web_application/src/hoc/user.js

```diff
@@ -0,0 +1,15 @@
+import { createSelector } from 'reselect';
+import { connect } from 'react-redux';
+import { UserSelector } from '../store/selectors/user';
+
+const mapStateToProps = createSelector(
+  [UserSelector],
+  userState => ({
+    user: userState.user,
+    isFetching: userState.isFetching,
```

Comment:
this will be a problem (it inherent to HoC generally):

if we use two HoC that provide the same property, it will be overwritten by the last one if I'm correct. 

It would be better to make a self-explanatory property like `isUserFetching`

Currently the best way to prevent this kind of error is to use a Render Prop instead of HoC cf. https://cdb.reacttraining.com/use-a-render-prop-50de598f11ce